petab/lint.py

```diff
@@ -621,6 +625,21 @@ def assert_noise_distributions_valid(observable_df: pd.DataFrame) -> None:
                     f"table: {distr}.")
 
 
+def assert_observable_id_is_unique(observable_df: pd.DataFrame) -> None:
+    """
+    Check if the observableId column of the observable table is unique.
+
+    Arguments:
+        observable_df: PEtab observable DataFrame
+
+    Raises:
+        AssertionError: in case of problems
+    """
+    if len(observable_df.index) != len(set(observable_df.index)):
+        raise AssertionError(
+            f"{OBSERVABLE_ID} column in observable table is not unique.")
+
```

Comment:
may be good to also return the duplicate ids.

Reply:
You mean to put the duplicated list inside the `AssertionError` message? It is certainly more informative. Again, if I do it here I should do it for `assert_parameter_id_is_unique` too I guess.

Reply:
not crucial, but I guess that would be helpful.